wiki/mnmg/Using_Infiniband_for_MNMG.md

```diff
@@ -0,0 +1,388 @@
+#   am_opt_zcopy_align: <= 512
+#         am_align_mtu: <= 4K
+#            am header: <= 127
+#               domain: device
+#           connection: to ep
+#             priority: 30
+#       device address: 3 bytes
+#           ep address: 4 bytes
+#       error handling: peer failure
+
+```
+
+You should also expect to see lines with `cuda_copy` and `cuda_ipc` transports:
+
+```
+#   Transport: cuda_copy
+#
+#   Device: cudacopy0
+#
+#      capabilities:
+#            bandwidth: 6911.00 MB/sec
+#              latency: 10000 nsec
+#             overhead: 0 nsec
+#            put_short: <= 4294967295
+#            put_zcopy: unlimited, up to 1 iov
+#  put_opt_zcopy_align: <= 1
+#        put_align_mtu: <= 1
+#            get_short: <= 4294967295
+#            get_zcopy: unlimited, up to 1 iov
+#  get_opt_zcopy_align: <= 1
+#        get_align_mtu: <= 1
+#           connection: to iface
+#             priority: 0
+#       device address: 0 bytes
+#        iface address: 8 bytes
+#       error handling: none
+```
+
+```
+# Memory domain: cuda_ipc
+#            component: cuda_ipc
+#             register: <= 1G, cost: 0 nsec
+#           remote key: 104 bytes
+#
+#   Transport: cuda_ipc
+#
+#   Device: cudaipc0
+#
+#      capabilities:
+#            bandwidth: 24000.00 MB/sec
+#              latency: 1 nsec
+#             overhead: 0 nsec
+#            put_zcopy: <= 1G, up to 1 iov
+#  put_opt_zcopy_align: <= 1
+#        put_align_mtu: <= 1
+#            get_zcopy: <= 1G, up to 1 iov
+#  get_opt_zcopy_align: <= 1
+#        get_align_mtu: <= 1
+#           connection: to iface
+#             priority: 0
+#       device address: 8 bytes
+#        iface address: 4 bytes
+#       error handling: none
+#
+
+```
+
+
+If you configured UCX with the `gdrcopy` option, you should also expect to see transports in this list:
+
+```bash
+# Memory domain: gdr_copy
+#            component: gdr_copy
+#             register: unlimited, cost: 0 nsec
+#           remote key: 32 bytes
+#
+#   Transport: gdr_copy
+#
+#   Device: gdrcopy0
+#
+#      capabilities:
+#            bandwidth: 6911.00 MB/sec
+#              latency: 1000 nsec
+#             overhead: 0 nsec
+#            put_short: <= 4294967295
+#            get_short: <= 4294967295
+#           connection: to iface
+#             priority: 0
+#       device address: 0 bytes
+#        iface address: 8 bytes
+#       error handling: none
+```
+
+To better understand the CUDA-based transports in UCX, refer to [this wiki](https://github.com/openucx/ucx/wiki/NVIDIA-GPU-Support) for more details. 
+
+
+## 2. Install ucx-py
+
+### From Conda
+
+Note: this package currently requires CUDA9.2. A CUDA10 package is in the works.
+
+`conda install -c conda-forge -c jakirkham/label/ucx cudatoolkit=9.2 ucx-py python=3.7`
+
+
+### From Source
+
+```bash
+git clone git@github.com:rapidsai/ucx-py
+cd ucx-py
+
+export UCX_PATH=$CONDA_PREFIX
+make -j install
+```
+
+## 3. Install NCCL
```

Comment:
Perhaps adding an additional note that nccl 2.4+ is required (so checking after the install that it is indeed 2.4+), that is the error we've been getting asked about the most